Description: Adds missing mapping for the minus char
 Under Firefox 21, the minus key on the alpha keyboard was missing, which was
 really irritating when typing commands on the SPICE console. This patch quite
 trivial patch fixes it.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: yes (by email)
Last-Update: 2013-05-25

--- spice-html5-0~0+20130318+ec9ae0cbd6.orig/utils.js
+++ spice-html5-0~0+20130318+ec9ae0cbd6/utils.js
@@ -208,6 +208,7 @@ common_scanmap[93]                 = 0xE
 
 /* Firefox/Mozilla codes */
 var firefox_scanmap = [];
+firefox_scanmap[173]                = KEY_Minus;
 firefox_scanmap[109]                = KEY_Minus;
 firefox_scanmap[61]                 = KEY_Equal;
 firefox_scanmap[59]                 = KEY_SemiColon;
